external/graphdb-service.js

```diff
@@ -18,7 +18,7 @@ class GraphdbService {
         this.logger = logger;
         this.logger.info(`Data repository name: ${this.config.repositoryName}`);
         const serverConfig = new ServerClientConfig('http://localhost:7200/')
-            .setTimeout(10000)
+            .setTimeout(40000)
```

Comment:
Let's use another approach here - I don't see this is a good approach (to wait up to 40s)

Reply:
If you know of any other approach. Let us know

Reply:
The timeout doesn't actually mean it's the time it takes for graph_db service to fully initialize. It actually takes around 15 secs for graph_db container to fully initialize. @kotlarmilos

Reply:
timeout lowered to 20sec